MakeHist/mumu.C

```diff
@@ -189,127 +202,122 @@ void mumu::Loop(TString sample_name)
+                               SBM1 = lep1_jet1_mass;
+                               SBM2 = lep2_jet2_mass;
+                           }
+                           else{
+                               SBM1 = lep2_jet1_mass;
+                               SBM2 = lep1_jet2_mass;
+                           }
+                       }
+                       //Fill max(SBM) and min(SBM) and |min(SBM)-173|
+                       //if(SBM1>170. || SBM2>170.){//>170 cut
+                       if(SBM1<SBM2){
+                           SBM_hist->Fill(SBM2,weight);
+                           mini_SBM_hist->Fill(SBM1,weight);
+                           float SBM_minus173 = SBM1 - 173;
+                           mini_SBM_minus173_hist->Fill(abs(SBM_minus173),weight);
                        }
-                       else{
-                           SBM1 = lep2_jet1_mass;
-                           SBM2 = lep1_jet2_mass;
+                       else {
+                           SBM_hist->Fill(SBM1,weight);
+                           mini_SBM_hist->Fill(SBM2,weight);
+                           float SBM_minus173 = SBM2 - 173;
+                           mini_SBM_minus173_hist->Fill(abs(SBM_minus173),weight);
                        }
 
-                       //Fill max SBM (Top mass bound)
-                       //if(SBM1>170. || SBM2>170.){//>170 cut
-                       if(SBM1<SBM2)SBM_hist->Fill(SBM2,weight);
-                       else SBM_hist->Fill(SBM1,weight);
 
                        //METvsMmm (Normalized MET)
                        float Mmm = ((leptons[0])+(leptons[1])).M();
-                       float MET_t = MET[0];
+                       float MET_t = met_pt;
                        float METvsMmm = MET_t/Mmm ;
                        //if(METvsMmm<0.2){ // <0.2 cut
                        METvsMmm_hist->Fill(METvsMmm,weight);
                        MET_hist->Fill(MET_t);
-                       dilep_mass_hist->Fill(Mmm);
+                       dilep_mass_hist->Fill(Mmm, weight);
 
                        //(HT-LT) calculation
                        float DHTLT = -999.;
-                       if(Nbjet > 1){
-                           DHTLT = (Jet1_B_PT+Jet2_B_PT) - (leptons[0].Pt()+leptons[1].Pt());
-                           //DHTLT = (All_bjet_PT) - (leptons[0].Pt()+leptons[1].Pt());
-                       }
-                       else if(Nbjet == 1){
-                           DHTLT = (bJets[0].Pt()+Jet2_nonB_PT) - (leptons[0].Pt()+leptons[1].Pt());
+                       if(Nbjet == 1 && Nnon_bjet == 0){
+                           DHTLT = (bJets[0].Pt()) - (leptons[0].Pt()+leptons[1].Pt());
                        }
-                       else if(Nbjet == 0){
-                           DHTLT = (Jet1_nonB_PT+Jet2_nonB_PT) - (leptons[0].Pt()+leptons[1].Pt());
+                       else {
+                           DHTLT = (Jet1_PT+Jet2_PT) - (leptons[0].Pt()+leptons[1].Pt());
```

Comment:
Why do we account for two leading jets and two leading leptons only?

Reply:
That's in line with what the paper did. It's also described in there. Maybe calling it H_T and L_T is a bit misleading. Technically it is and always was H_T(2) and L_T(2)

Reply:
Yes. I actually wrote the codes using the paper and Delphes study codes.